src/functions-and-arrays.js

```diff
@@ -52,14 +97,31 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(something) {
+  if (something.length === 0){
+    return null;
+  }
+  let unique = [...new Set(something)];
+  return unique;
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(something, starting) {
```

Comment:
Sorry, naming again :). It would be better to name it something like (arr, word), or (haystack, needle) if you'd like, just something explicit.